AUTHORS

```diff
@@ -61,6 +61,7 @@ source{d} <hello@sourced.tech>
 Shawn Smith <shawnpsmith@gmail.com>
 Spencer Lyon <spencerlyon2@gmail.com>
 Steve McCoy <mccoyst@gmail.com>
+Takeshi Yoneda <cz.rk.t0415y.g@gmail.com>
```

Comment:
If you are welcome, i want to continue to support and contribute to gonum projects.

Reply:
That would be good.